### PR TITLE
Edits in CONTRIBUTING.md Section 3.1.b #7134

### LINKED ISSUE #7134: Edits to `CONTRIBUTING.md` at Pull Requests, Sec 3.1.b
### Overview
We are making edits to the actual `pull-request-template.md`. The instructions in the `CONTRIBUTING.md` file need to match so that there is no unnecessary ambiguity.
### Action Items
- [x] Open [CONTRIBUTING.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md). 
- [x] Note: UPDATED 8/14/24: Go to Section 3.1.b, then find the code block for the pull request markdown template. Replace the existing code with:  

  ````  
   ```
  <!--  Important! Add the number of the issue you worked on  --> 
  Fixes #replace_this_text_with_the_issue_number

  ### What changes did you make?
  <!--  Note: add lines if needed, and remove any unused lines  -->  
    - 
    - 
    - 

  ### Why did you make the changes (we will use this info to test)?
  <!--  Note: add lines if needed, and remove any unused lines  -->  
    - 
    - 
    - 

  ### Screenshots of Proposed Changes To The Website (if any, please do not include screenshots of code changes)
  <!--  Notes: 
    - If there are no visual changes to the website, delete all of the script below and replace with "- No visual changes to the website"
    - If there are visual changes to the website, include the 'before' and 'after' screenshots below. 
    - If your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images
    - If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag 
   --> 

  <details>
    <summary>Visuals before changes are applied</summary>

    ![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

  </details>

  <details>
    <summary>Visuals after changes are applied</summary>

    ![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

  </details>
  ```
  ````  
- [x] Note! Do not make any other changes besides the change to this full pull request template in Section 3.1.b. For example, do not change the snippets / excerpts of the pull request template in Sections 3.1.b.i through 3.1.b.iv. 
- [x] Changes to `CONTRIBUTING.md` cannot be tested locally, rather they must be tested after pushing the issue branch to your fork of the repository. Push your issue branch in the usual manner, but before creating the Pull Request, check your updates using this test URL, replacing "your github", "your repo", and "your branch" accordingly.
  ```
  https://github.com/{your github}/{your repo}/blob/{your branch}/CONTRIBUTING.md
  ```
   for example: 
  ```
  https://github.com/roslynwythe/website/blob/test-CONTRIB/CONTRIBUTING.md
  ```
- [x] Create a pull request with your changes. In the Pull Request, after the "Why did you make the changes" section, add this line to help reviewers, replacing the text in brackets (and the brackets) with the test URL from the previous Action Item.  
  ```
  For Reviewers: Do not test changes locally, rather test changes at [REPLACE WITH TEST URL]
  ```

### Resources/Instructions
- This issue was initiated by #6929

### PR DESCRIPTION
**Pull request by patelbansi3009**
Fixes #7134 

### What changes did you make?
  - Updated code which starts on Line 825 and ends on Line 855.
  - Edited the example of the pull-request-template.md in the CONTRIBUTING.md file

### Why did you make the changes (we will use this info to test)?
  - The instructions in the CONTRIBUTING.md file need to match so that there is no unnecessary ambiguity.
  - For Reviewers: Do not test changes locally, rather test changes at https://github.com/patelbansi3009/website/blob/edits-in-CONTRIBUTING.md-section-3.1.b-7134/CONTRIBUTING.md

### Screenshots of Proposed Changes Of The Website  (if any, please do not screenshot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>

image

</details>

### CODE REVIEW

**Comment by dcotelessa** [comment]
One of the PR checks is failing. Can we describe why this is faling and then fix?
Thanks.
"Set PR Labels / generate-labels-artifact (pull_request) Failing after 8s"

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -823,21 +823,28 @@ git push --set-upstream origin update-give-link-2093
   <summary><strong>Click here</strong> to see pull request markdown template</summary>
 
   ```
+  <!--  Important! Add the number of the issue you worked on, for example: 'Fixes #9876'   -->
```
Comment (t-will-gillis) [inline CONTRIBUTING.md]: Hey @patelbansi3009 This is not your error, however please replace line 826 above with:
  ```  
  <!--  Important! Add the number of the issue you worked on  --> 
  ```

**Review by t-will-gillis — CHANGES_REQUESTED** [review]
Hey @patelbansi3009 

As @dcotelessa pointed out, there is a failing PR check. This is not because of anything you did, rather it is due to an unintended conflict with the text in the original issue (which I wrote-ugh).*** Please substitute the new text as shown. Otherwise, your branches look good and you are linking the original issue. For "What changes did you make?", I would recommend simplifying this by removing the actual code changes, and instead stating something to the effect of: "Edited the example of the `pull-request-template.md` in the `CONTRIBUTING.md` file". Otherwise, looks good!

***FYI the reason that "Set PR Labels" workflow is failing is because the text for the example PR template includes the line `... for example 'Fixes # 9876'`. When your PR was created, the bot did a regex search for an issue number appearing after the pattern "Fixes #' and likely found both the correct issue number reference (7134) and the template example (9876). The workflow then crashed when it could not find the example issue nor retrieve its labels. My bad.

File: CONTRIBUTING.md (hunk 1 above)
Comment (dcotelessa) [reply CONTRIBUTING.md]: Good to know, thanks, @t-will-gillis

Comment (patelbansi3009) [reply CONTRIBUTING.md]: Hey @t-will-gillis , I have updated the code and edited the PR itself. Thank you for the help!

**Comment by patelbansi3009** [comment]
Hey @dcotelessa and @t-will-gillis , Sorry for the delayed update, Please check the PR and let me know if any updates need to be done. Thank You!

**Review by t-will-gillis — APPROVED** [review]
Hey @patelbansi3009 Thank you for making the change. The automations re-ran when you resubmitted and they all went through. 
- Branches look good
- There is a link back to the original issue
- There are concise descriptions of what was done and why
- The screenshots are good- even though they aren't website changes, I feel like it is appropriate to show the `CONTRIBUTING.md`
- The code changes are clean  

Thank you for working on this!

**Comment by jphamtv** [comment]
Hi @dcotelessa and @pluto-bell, Could you please provide your estimated time for review and your availability? Thanks!

**Comment by dcotelessa** [comment]
ETA:  Thu-Fri: 3:30-5:30pm
Availability: 1 hr.

**Review by dcotelessa — APPROVED** [review]
Thanks for your fixes!

- The precheck is working now.
- Branch is named well
- Link to issue
- Well written explanations
- I like the screenshots of the changes just to confirm the differences

Great job!

**Pull request merged**